tests/tls_challenge_test.go

```diff
@@ -0,0 +1,258 @@
+//go:build test_e2e
+
+package tests
+
+import (
+	"context"
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"os"
+	"regexp"
+	"testing"
+	"time"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/docker/config"
+	"github.com/stackrox/rox/pkg/namespaces"
+	"github.com/stretchr/testify/suite"
+	appsV1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	apiErrors "k8s.io/apimachinery/pkg/api/errors"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+const (
+	s                  = namespaces.StackRox // for brevity
+	proxyNs            = "qa-tls-challenge"  // Must match the additionalCA X509v3 Subject Alternative Name
```

Comment:
In multiple functions we shadow this constant. Why? If it's a constant why not just use it?

Reply:
Good point, let me follow up on this in https://github.com/stackrox/stackrox/pull/12305 for the sake of progress.